packages/components/src/local/molecules/json-editor/index.tsx

```diff
@@ -74,19 +73,25 @@ export const JsonEditor: React.FC<Props> = ({
     return memoryName
   }
 
+  const getEditLocalStorgaeid = (): string => `edit-${genLocalStorageId()}`
+
   const OnSave = () => {
     saveMessage && saveMessage()
     expiredStorage.removeItem(genLocalStorageId())
+    expiredStorage.removeItem(getEditLocalStorgaeid())
   }
 
   const onPopupCancel = (): void => {
-    expiredStorage.removeItem(genLocalStorageId())
+    // removePlanning
```

Comment:
@lilitkarapetyan  - do you remember what the `removePlanning` comment is for?

Reply:
Should we delete the chached message ?